src/Callout/Callout.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+import React, { type Element, type Node } from 'react';
+import { Flex } from '../primitives';
+import _Callout, { CalloutHeader } from './styled';
+import { getUniqueId } from '../uniqueId';
+
+import CalloutContent from '../Card/CardContent';
```

Comment:
We need to make these themeable, so let's put them in `./styled`. This will also help when we want to create the destructured version of `<Callout>`.